transfermethodui/src/main/java/com/hyperwallet/android/ui/transfermethod/view/SelectTransferMethodFragment.java

```diff
@@ -501,18 +502,37 @@ void bind(TransferMethodSelectionItem selectionItem) {
                 if (isFeeAvailable(selectionItem.getFees())
                         && isProcessingTimeAvailable(selectionItem.getProcessingTime())) {
                     mDescriptionFeesAndProcessingTime.setVisibility(View.VISIBLE);
-                    mDescriptionFeesAndProcessingTime.setText(mDescriptionFeesAndProcessingTime.getContext()
-                            .getString(R.string.feeAndProcessingTimeInformation, formattedFee,
-                                    selectionItem.getProcessingTime().getValue()));
+                    if (isZeroFeeAvailable(mDescriptionFeesAndProcessingTime.getContext(), formattedFee)) {
```

Comment:
same applies in this method too, too many if/else conditions, please refactor

Reply:
move to common method in FeeFormatter as getFormattedFeeAndProcessingTime()